README.md

```diff
@@ -48,100 +48,127 @@ the reader to fulfill an action.
 
 # Trying Out Konflux
 
-This section demonstrates the process for deploying Konflux locally, onboarding users
+This section demonstrates the process for deploying Konflux locally, onboarding users,
 and building and releasing an application. The procedure contains two options for the
 user to choose from for onboarding applications to Konflux:
 
 - Using the Konflux UI
 - Using Kubernetes manifests
 
-Each of those options has its pros and cons: the procedure described using the UI,
-provides more streamlined user experience once setup is done, but it requires using
-[Quay.io](https://quay.io) for image registry and requires some additional initial setup
-steps comparing to using Kubernetes manifest alone. The latter also supports using any
-image registry.
+Each of these options has its pros and cons. The Konflux UI procedure provides a more
+streamlined user experience once you finish installing and configuring it. However,
+it requires using [Quay.io](https://quay.io) for image registry, plus some additional
+initial setup steps compared to using a Kubernetes manifest alone.
 
-**Note:** The procedure that is described using the UI can also be fulfilled using CLI
-and Kubernetes manifests.
+**Note:** You can configure your server to operate the same way using either the
+Konflux UI method or the terminal and Kubernetes manifests procedure. For example,
+you can also configure Konflux to support an image registry using the terminal process.
 
-In both cases, the recommended way to try out Konflux is using
-[Kind](https://kind.sigs.k8s.io/).
-The process below creates a Kind cluster using the provided config in this repository.
-The config tells Kind to forward port `9443` from the host to the Kind cluster. The port
-forwarding is needed for accessing Konflux.
+In both cases, the recommended way to try out Konflux is using [Kind](https://kind.sigs.k8s.io).
```

Comment:
We're keeping line length at 89 characters. That's the reason for the end of this line to be pushed to the next line.

Reply:
I commented the same for some other lines, but there are more.